### ISSUE TITLE
Can't locate scenarios, MoM, MacOS

### ISSUE THREAD

**Issue by kalgarion** [NPBruce/valkyrie#450]
Hi,

I have downloaded the Valkyrie app and it runs fine, but I can't access any scenarios through the Start Scenario/Download option. I get a black screen with the top logo "Download Scenario" and a red "Back" icon in the bottom left corner. I don't know what is supposed to happen, but if a field to enter a download path or something is supposed to appear this does not happen.

As a workaround I tried to download the scenarios manually from github, but no luck their either. What files do I need to download? I downloaded for example "ExoticMaterial.valkyrie" and put the file under .config/Valkyrie but it still isn't visible in the app Select Scenario menu.

To fault-find I created my own scenario using the Scenario Editor, and this seem to work  (it becomes visible in the Select Scenario menu). I see in .config/Valkyrie that my created scenario is not stored as a .valkyrie file, but instead as a folder with the scenario name and a content of some .ini and .txt files. I guess I am just blind or stupid, but where do I find such bundles (zipped?) here on github, to download the available scenarios (e.g Exotic Materials, The Fall of House Lynch etc)?

Regards,
Jon

**Comment by NPBruce** [comment]
For clarity: This is MacOS and I don't know what is going on.

.valkyrie packages are just renamed zips, it shouldn't make any difference if the files are packaged are not (you can try extracting the .valkyrie files if you like).

**Comment by kalgarion** [comment]
Hmm. Ok. If I manually rename the .valkyrie file and unzip it I indeed get a folder structure similar to the ones I get when creating a custom scenario using the in-app editor. But for some reason only the in-app created scenarios are visible in Select Scenario. 

I also tried to copy-paste all the .ini and .txt files from one of the downloaded, unzipped scenarios into my created scenario (EditorScenario1), but this just removed it completely from the Select Scenario menu.
